main/exercise/ptest_agree_reorder.class.php

```diff
@@ -0,0 +1,296 @@
+<?php
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)